ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
     box-sizing: border-box;
```

Comment:
The `min-height` and `max-height` properties have been removed. Could you explain the reasoning behind this change? It might affect the panel's layout and responsiveness.